### PR TITLE
make some changes of remember version 1

### PR DESCRIPTION
**Pull request by Aquafina99**
@woodenchalet Please view my code and feel free to add any comment that could help me to make this version better. Thanks!
I added some Here

### CODE REVIEW

File: RememberV1.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+time.sleep(2.5)
+print('sandwich')
+time.sleep(2.5)
+
+PlayerAnswer = input('What word began with letter c?')
+
+print('Congratulation, you are correct.')
+print('Sorry, you selected ' + PlayerAnswer)
+print(' The current answer was ' + 'Chair')
+input('Press enter to close the window.')
```
Comment (woodenchalet) [inline RememberV1.py]: Add a line after the last line.

**Comment by Aquafina99** [comment]
@woodenchalet Please view my code and feel free to add any comment that could help me to make this version better. Thanks!

File: RememberV1.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+__author__ = 'ceciliaxiang'
+# Remember Version 1
+# In this game, the player will try to remember
+# as many as words that game displays
+# and input the word began with particular letter
+
```
Comment (woodenchalet) [inline RememberV1.py]: Learn the formal way to comment the code description, date, author.

**Comment by woodenchalet** [comment]
Added some comments, thanks.

**Pull request merged**